nvchecker_source/cran.py

```diff
@@ -0,0 +1,16 @@
+CRAN_URL = 'https://cranchecks.info/pkgs/%s'
+
+
+async def get_version(name, conf, *, cache, **kwargs):
+    key = conf.get('cran', name)
+    data = await cache.get_json(CRAN_URL % key)
+    flavors = [
+        "r-devel-linux-x86_64-debian-clang",
+        "r-devel-linux-x86_64-debian-gcc",
+        "r-devel-linux-x86_64-fedora-clang",
+        "r-devel-linux-x86_64-fedora-gcc",
+        "r-release-linux-x86_64"
+    ]
```

Comment:
What about Windows and macOS?

Reply:
we don't create pkg for Windows and macOS.

Reply:
nvchecker is not only for lilac.

Reply:
should I raise an Error or something?

Reply:
I don't really know what "flavors" mean here. Is it possible for a cran package to have different versions for different flavors?

Reply:
`flavor` means different platform/system (windows, macos, linux), arch (x86_64, arm), compiler (gcc, clang). It might be a little chance that there is different versions for different flavors.